Remotion/Validation/UnitTests/RuleBuilders/AddingPropertyValidationRuleBuilderTest.cs

```diff
@@ -40,10 +39,14 @@ public class AddingPropertyValidationRuleBuilderTest
     public void SetUp ()
     {
       _addingPropertyValidationRuleCollectorMock = new Mock<IAddingPropertyValidationRuleCollector>(MockBehavior.Strict);
-      _addingPropertyValidationRuleCollectorMock.Setup(stub => stub.Property).Returns(PropertyInfoAdapter.Create(typeof(Customer).GetProperty("UserName")));
+      _addingPropertyValidationRuleCollectorMock
+          .Setup(stub => stub.Property)
+          .Returns(PropertyInfoAdapter.Create(typeof(Customer).GetProperty("UserName")));
```

Comment:
```suggestion
          .Returns(PropertyInfoAdapter.Create(MemberInfoFromExpressionUtility.GetProperty((Customer _) => _.UserName)));
```
```suggestion
          .Returns(PropertyInfoAdapter.Create(MemberInfoFromExpressionUtility.GetProperty((Customer _) => _.UserName)));
```
Info:
This syntax is typesafe. I prefer to use it with new code but we do have enough older code, too, so it doesn't stand out as "strange" when it is used.